web/elm/tests/BuildTests.elm

```diff
@@ -3,7 +3,7 @@ module BuildTests exposing (all)
 import Array
 import Build
 import Build.Models as Models
-import Build.Msgs as Msgs
+import Build.Msgs
```

Comment:
Is this the direction we want to go in general? always explicitly prefixed messages in tests?

Reply:
I'm still hopeful that in the long run our messages will be very flat types and there will be only one Msgs module... Are you thinking the alternative here would be to expose `Msg(..)` in the test? I don't really have a strong opinion on that one

Reply:
No, just `Build.Msgs` vs `Build.Msgs as Msgs`. I think exposing would make it hard to read.